### DISCUSSION TITLE
Hernoemen afschot grafiek en tabel

### DISCUSSION THREAD

**Discussion by SanderDevisscher** [inbo/reporting-rshiny-grofwildjacht#359] [Polls]
Niko haalt aan dat de titels van Evolutie gerapporteerd afschot en Gerapporteerd afschot een verwarrende combo is. Hij stelt voor om de titels te veranderen naar resp. `Totaal gerapporteerd afschot` en `(Details) gerapporteerd afschot`

@RuttenAnneleen, @janvercammen, @emiliegelaude, @jimcasaer, @LynnINBO, @AxelNeukermans  wat denken jullie ?

**Comment by AxelNeukermans** [comment]
👍

*Mvg*

*Axel Neukermans*

Deskundige Team Faunabeheer en Invasieve Soorten

Instituut voor Natuur- en Bosonderzoek (INBO)

Gaverstraat 4, 9500 Geraardsbergen, België

https://orcid.org/0000-0003-0272-9180


gsm +32 478 880 791

*www.inbo.be <http://www.inbo.be/>*








Op di 22 nov. 2022 om 14:36 schreef Sander Devisscher <
***@***.***>:

**Comment by RuttenAnneleen** [comment]
Dan zou ik gaan voor 'Evolutie gerapporteerd afschot' en '(Details) gerapporteerd afschot'
Maar voor mij zou het ook goed zijn om het te houden of veranderen zoals Niko voorstelt :)
